src/crimson/osd/shard_services.h

```diff
@@ -113,9 +113,15 @@ class PerShardState {
   bool stopping = false;
   seastar::future<> stop_registry() {
     assert_core();
-    crimson::get_logger(ceph_subsys_osd).info("PerShardState::{}", __func__);
+    auto& logger = crimson::get_logger(ceph_subsys_osd);
```

Comment:
Please use SET_SUBSYS and DEBUG

Reply:
I think with current declaration of SOURCE_SUBSYS in log.h we can;t use SET_SUBSYS in header file. It is giving multiple definition error. First we have to handle it log.h then we can use it. We can handle it via separate patch not this patch.